packages/runtime/src/helpers/utils.ts

```diff
@@ -128,18 +143,38 @@ export const redirectsForNextRouteWithData = ({
     force,
   }))
 
-export const getApiRewrites = (basePath) => [
-  {
-    from: `${basePath}/api`,
-    to: HANDLER_FUNCTION_PATH,
-    status: 200,
-  },
-  {
-    from: `${basePath}/api/*`,
-    to: HANDLER_FUNCTION_PATH,
-    status: 200,
-  },
-]
+export const getApiRewrites = (basePath: string, apiRoutes: Array<ApiRouteConfig>) => {
+  const apiRewrites = apiRoutes.map((apiRoute) => {
+    const [from] = toNetlifyRoute(`${basePath}${apiRoute.route}`)
+
+    // Scheduled functions can't be invoked directly, so we 404 them.
+    if (apiRoute.config.type === 'experimental-scheduled') {
+      return { from, to: '/404.html', status: 404 }
+    }
+    return {
+      from,
+      to: `/.netlify/functions/${getFunctionNameForPage(
+        apiRoute.route,
+        apiRoute.config.type === 'experimental-background',
+      )}`,
+      status: 200,
+    }
+  })
+
+  return [
+    ...apiRewrites,
+    {
+      from: `${basePath}/api`,
```

Comment:
I’m clearly missing something, but I’m not understanding  why this rewrite is necessary if if we have a rewrite for `/api/*?` is something being hitting at `api`?